src/graphql/operations/users.ts

```diff
@@ -8,36 +8,52 @@ export default async function (parent, args) {
 
   checkLimits(args, 'users');
 
-  const fields = {
-    id: 'string',
-    ipfs: 'string',
-    created: 'number'
-  };
-  const whereQuery = buildWhereQuery(fields, 'u', where);
-  const queryStr = whereQuery.query;
-  const params: any[] = whereQuery.params;
-
-  let orderBy = args.orderBy || 'created';
-  let orderDirection = args.orderDirection || 'desc';
-  if (!['created'].includes(orderBy)) orderBy = 'created';
-  orderBy = `u.${orderBy}`;
-  orderDirection = orderDirection.toUpperCase();
-  if (!['ASC', 'DESC'].includes(orderDirection)) orderDirection = 'DESC';
+  const fields = { id: 'string', ipfs: 'string', created: 'number' };
```

Comment:
There is no way to filter with space?

Reply:
a `space` filter does not make sense on this `User` object. Maybe you're talking about Leaderboard ?

Reply:
oh yes I mean leaderboard 😄 why do we want to filter with vote_count and proposal_count but not space? 🤔

Reply:
Filter by vote_count to for example fetch only user who have vote_count < X

Filterting by space will seem weird, like "I want to filter users with space `abc.eth`". What would that filter mean ?